awsagentprovider/src/main/java/software/amazon/opentelemetry/javaagent/providers/AwsSpanProcessingUtil.java

```diff
@@ -226,6 +227,7 @@ private static String generateIngressOperation(SpanData span) {
   static boolean isDBSpan(SpanData span) {
     return isKeyPresent(span, DB_SYSTEM)
         || isKeyPresent(span, DB_OPERATION)
+        || isKeyPresent(span, DB_USER)
```

Comment:
I don't think we agreed to this. I think our definition of `isDbSpan` is still: `DB_SYSTEM`, `DB_OPERATION`, and/or `DB_STATEMENT`, why is this changing?

Reply:
Sure, will update this.